coffee-backend/src/main/kotlin/chat/letscoffee/security/repository/UserRepository.kt

```diff
@@ -0,0 +1,12 @@
+package chat.letscoffee.security.repository
+
+import chat.letscoffee.security.model.User
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+interface UserRepository : JpaRepository<User?, Long?> {
+    fun findByEmail(email: String?): Optional<User>
```

Comment:
`Optional` is (almost) the equivalent to a `User?` in Kotlin. We should stick to Kotlin practices and return a `User?` instead

Reply:
I think I read somewhere that Optional was still preferred when using JPA. Have you worked as this before? 🌋